src/routes/publicRoutes.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+const publicRoutes = [<Route exact path="/" />, <Route exact path="/megaStrona" />, <Route exact path="/signIn" />];
+
+export { publicRoutes };
```

Comment:
export { publicRoutes }; 

{} is not necessary

or 

```export const publicRoutes = [<Route exact path="/" />, <Route exact path="/megaStrona" />, <Route exact path="/signIn" />];``` 

Refers to all files affected by similar change